src/inverted-index.js

```diff
@@ -6,99 +6,110 @@
  * @class
  */
 class InvertedIndexClass {
-    /**
-     * Class Instantiation.
-     */
+  /**
+   * Class Instantiation.
+   */
   constructor() {
     this.index = {};
     this.books = {};
     this.parsedBooks = {};
   }
-    
-    /**
-     * create Index
-     * 
-     * creates an index of words
-     * 
-     * @param{string}  filename accepts a string name for a file;
-     * @param{object} books is a json object for which an index is to be created
-     * @returns{object};
-     */
+
+  /**
+   * create Index
+   * 
+   * creates an index of words
+   * 
+   * @param{string}  filename accepts a string name for a file;
+   * @param{object} books is a json object for which an index is to be created
+   * @returns{object};
+   */
   createIndex(filename, books) {
     this.index[filename] = {};
-    this.books = books;
-
-    for (let i = 0; i < books.length; i += 1) {
-      const textIndex = books[i].text.toLowerCase().match(/[\w']+/g);
-      for (let j = 0; j < textIndex.length; j += 1) {
-        if (this.index[filename][textIndex[j]] === undefined) {
-          this.index[filename][textIndex[j]] = {};
-          this.index[filename][textIndex[j]][i] = true;
+    this.books = (books);
+    let bookIndex;
+    books.forEach((singleBook, inex) => {
+      // console.log(singleBook);
+      singleBook = `${singleBook.title} ${singleBook.text}`;
+      singleBook = singleBook.toLowerCase().match(/[\w]+/g);
+      bookIndex = inex;
+      singleBook.forEach((term) => {
+        if (this.index[filename][term] === undefined) {
+          this.index[filename][term] = {};
+          this.index[filename][term][bookIndex] = true;
         } else {
-          this.index[filename][textIndex[j]][i] = true;
+          this.index[filename][term][bookIndex] = true;
         }
-      }
-    }
+      });
+    });
     return this.index;
   }
+
   /**
-     * returns created index
-     * 
-     * @returns{object};
-     */
+   * returns created index
```

Comment:
### Nitpick

The rest of the doc blocks have the name of the method, why is this one exempt?